project/Dependencies.scala

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka
+
+import sbt._
+import Keys._
+
+object Dependencies {
+  val Scala211 = "2.11.12"
+  val Scala212 = "2.12.17"
+  val Scala213 = "2.13.10"
+
+  val buildScalaVersion = System.getProperty("akka.build.scalaVersion", Scala212)
+
+  val AkkaManagementVersion = "1.0.5"
+  // play 2.6 requires 10.0.15
+  val AkkaHttpVersion_10_0 = "10.0.15"
+  val AkkaHttpVersion_10_1 = "10.1.11"
+
+  val commonsLang = "org.apache.commons" % "commons-lang3" % "3.5" // ApacheV2
+
+
+  val streamContrib =                "com.typesafe.akka" %% "akka-stream-contrib"                 % "0.10"                          // ApacheV2
+
+  lazy val akkaVersion = settingKey[String]("The version of Akka to use.")
+  lazy val akkaHttpVersion = settingKey[String]("The version of Akka HTTP to use.")
+
+  val Versions = Seq(
+    akkaVersion := version213(Akka.version27, Akka.version).value,
+    akkaHttpVersion := version213(AkkaHttpVersion_10_1, AkkaHttpVersion_10_0).value,
+  )
```

Comment:
Can I just use Akka version `2.7.0` and Akka HTTP `10.1.11` or we still need this flexibility?

Reply:
no need for the alternatives. You can use Akka 2.7.0 and HTTP **10.4.0**

Reply:
removed